src/ui/components/tenants/tenantDetail/thirdPartyPage/ThirdPartyPage.tsx

```diff
@@ -88,6 +87,11 @@ const ProviderInfo = ({
 				setProviderConfigResponse(response.providerConfig);
 			}
 		} catch (error) {
+			showToast({
+				iconImage: getImageUrl("form-field-error-icon.svg"),
+				toastType: "error",
+				children: <>Something went wrong please try refreshing the page</>,
```

Comment:
Right now since we throw an error during render [in this case](https://github.com/supertokens/dashboard/blob/77b151ffb4c74eb122c9eed6a3afc4266a8739bb/src/ui/components/tenants/tenantDetail/thirdPartyPage/ThirdPartyPage.tsx#L126), so it would show the error boundary screen.